lib/derivative_rodeo/generators/thumbnail_generator.rb

```diff
@@ -10,7 +10,7 @@ class ThumbnailGenerator < BaseGenerator
       # We want to mirror the same file "last" extension as described in Hyrax.
       #
       # @see https://github.com/samvera/hyrax/blob/426575a9065a5dd3b30f458f5589a0a705ad7be2/app/services/hyrax/file_set_derivatives_service.rb
-      self.output_extension = 'thumbnail.jpg'
+      self.output_extension = 'thumbnail.jpeg'
```

Comment:
Reading note: we will need to coordinate the output extension for thumbnails; but this is adequate.